en/flight_controller/ocpoc_zynq.md

```diff
@@ -41,11 +41,12 @@ After setting up the PX4 development environment and cloning the [PX4 Firmware](
 
   * For Aerotenna's default ubuntu-armhf root file system:
     ```
-    make posix_ocpoc_ubuntu
+    make aerotenna_ocpoc_ubuntu
     ```
   * For Petalinux:
     ```
-    make posix_ocpoc_cross
+    make posix_ocpoc_cross 
```

Comment:
@dagar What is equivalent for this case?

Reply:
Thanks, that was actually covered in the previous line. I've removed this in https://github.com/PX4/px4_user_guide/pull/392